gcc/tree-core.h

```diff
@@ -1364,6 +1364,9 @@ struct GTY(()) tree_base {
        ENUM_IS_OPAQUE in
 	   ENUMERAL_TYPE
 
+      EXPR_CONTRACT_CONST_WRAPPER_P in
```

Comment:
We should use a function name in the comment, not a macro name, the macros are.. ..going away. Like contract_semantic is mentioned for various tree flags in cp-tree.h.